LondonBritishLibrary/orient/BLorient3663.xml

```diff
@@ -0,0 +1,152 @@
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <title ref="LIT7094PrayerDevils">Prayer against devils, Bāryā, Legewon, migraine, colic, chest pain, 
+                                rheumatism, Budā, Falašā, Gudale, Dabbās, and all diseases</title>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> በስሙ፡ ለአብ፡ በስሙ፡ ለወልድ፡ በስሙ፡ ለመንፈስ፡ ቅዱስ፡ ታኦስ፡ አዝያስ፡ ሚልክያስ፡ 
+                                <gap reason="ellipsis"/> በዝ፡ አስማት፡ ስድዶሙ፡ ለአጋንንት፡ ለባርያ፡ ወለሌጌዎን፡ ለፍልፀት፡ ለቁርፀት፡ ለውግዓት፡ ወልቁርጥማት፡ ለቡዳ፡ ወፈላሻ፡ ለጉዳሌ፡
+                                ለደባስ፡ እምኵሉ፡ ሕማም፡ እቀቦ፡ ለገብርከ፡ 
+                                <persName role="owner" ref="PRS14548WaldaAragawi">ወልደ፡ አረጋ<supplied reason="undefined" resp="PRS8999Strelcyn">ዊ</supplied>፡</persName></incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <title ref="LIT5696MagicPrayer"/>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ጸሎት፡ ዘሐፁረ፡ መስቀል፡ መግረሬ፡ ፀር፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <title ref="LIT2765RepCh49"/>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ሰዳዴ፡ አጋንንት፡ ፋኑኤል፡ ለእግዚአብሔር፡ እምጽርሑ፤</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i5">
+                            <title ref="LIT7095PrayerColic">Prayer against colic</title>
+                            <note>Containing ancient or irregular form of the letter <foreign xml:lang="gez">ዥ</foreign> with four small strokes 
+                                marking the sixth order: 
+                                <figure><graphic url="https://betamasaheft.eu/resources/images/snippets/BLorient12996/sign2.png" height="30px">
+                                </graphic></figure></note>
+                            <incipit xml:lang="gez">ጸሎት፡ ዘቍርፀት፡ እልፍዑል፡ ፫ዘይጐድጓድ፡ ገበዋተ፡ ወይ<gap reason="illegible"/> አንዥተ፡ ወይሰብር፡ 
+                                አእ<supplied reason="undefined">ም</supplied>ጽምተ፡ ወይቈርጥ፡ አባላተ፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Scroll">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">1</measure>
```

Comment:
I wonder if the measure is 3 strips, and not 1 leaf

Reply:
If I remember it right, all scrolls, that I have encoded so far, have been encoded as one leaf. The dimension refer to the scroll altogether but not to the single strips.